search.yaml

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) 2017, the Dart project authors.  Please see the AUTHORS file
+# for details. All rights reserved. Use of this source code is governed by a
+# BSD-style license that can be found in the LICENSE file.
+
+runtime: custom
+env: flex
+service: search
+env_variables:
+  # Needs to be in sync with lib/shared/configuration.dart
+  DUCENE_DIR: '/ducene'
+
+resources:
```

Comment:
Not sure what kevin means, but the cpu/memory_gb just get rounded up sometimes. It's fine to specify our minimum requirements, to make it explicit how much cpu/ram we're guaranteed to get.

It might be better to double the cpu/ram here.